tools/standard-build/standard-build.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && \
+    apt-get install -y openjdk-14-jdk && \
+    apt-get install -y curl && \
+    apt-get install python -y && \
+    apt-get install git -y
+
+# Create user and group
+RUN groupadd -g 1000 opensearch && \
+    useradd -u 1000 -g 1000 -d /usr/share/opensearch opensearch && \
+    mkdir -p /usr/share/opensearch && \
+    chown -R 1000:1000 /usr/share/opensearch
```

Comment:
Can we make the GID and UID configurable (with a sensible default)? Similar to https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/main/elasticsearch/docker/dockerfiles/AL2.dockerfile

Reply:
Made the change. Can you review if again please? Thanks!